src/api/app/models/token/workflow.rb

```diff
@@ -16,6 +16,8 @@ def call(options)
     workflows.each(&:call)
   rescue Octokit::Unauthorized, Gitlab::Error::Unauthorized => e
     raise Token::Errors::SCMTokenInvalid, e.message
+  ensure
+    set_triggered_at
```

Comment:
I would also suggest setting the triggering time right after the token is called, not after the whole `.call`method is performed.

For example, if the token needs to run many workflows with a big list of steps, the difference between the time when the user really triggered the token and the one we set in the database can be significant.

What do you think @krauselukas @dmarcoux @vpereira ?

Reply:
Edit: "... right after/before the token is called"

Reply:
Alright, I will change the approach

Reply:
This is addressed, setting the now the date and time right at the beginning of the token call